portfolio/src/main/webapp/index.html

```diff
@@ -40,6 +40,15 @@ <h2>About Me </h2>
   <p>Hi There, I am Prashant Neupane all the way from Nepal. Nepal is a small
       landlocked country in the Himalayas. I go to Fisk University in Nashville, TN.
       I am a Computer Science Major and rising Junior.
+  <button onclick="showComments()">Show Commnets!</button>
+  <div id="greeting-container"></div>
+  <div class="card">
+    <div class="card-header">
+      COMMENTS
+    </div>
+    <ul id="comments" class="list-group list-group-flush">
+    </ul>
+  </div>
```

Comment:
Use HTML5 elements `<section>` and `<header>` (or one of the header elements, e.g. `h3`) here for clarity. They behave exactly like `<div>`s do.

```html
<section class="card">
  <header class="card-header">COMMENTS</header>
  <ul id="comments" class="list-group list-group-flush"></ul>
</section>
```

Reply:
Thanks for the edits
Changes made